_includes/homepage-features-icon-band.html

```diff
@@ -6,10 +6,22 @@ <h2>Quarkus Benefits</h2>
   </div>
   <div class="grid-container">
     <div class="width-4-12 width-12-12-m contrib-block">
-      <img class="light-only" src="{{site.baseurl}}/assets/images/about/icon-containerfirst.svg">
-      <img class="dark-only" src="{{site.baseurl}}/assets/images/about/icon-containerfirst-dark.svg">
-      <h3><a href="/container-first">Container First</a></h3>
-      <p>Quarkus tailors your application for GraalVM and HotSpot. Amazingly fast boot time, incredibly low RSS memory (not just heap size!) offering near instant scale up and high density memory utilization in container orchestration platforms like Kubernetes. We use a technique we call compile time boot.</p>
+      <img class="light-only" src="{{site.baseurl}}/assets/images/performance/icon-performance.svg">
+      <img class="dark-only" src="{{site.baseurl}}/assets/images/performance/icon-performance-dark.svg">
+      <h3><a href="/performance">Performance</a></h3>
+      <p>Quarkus streamlines framework optimizations in the build phase to reduce runtime dependencies and improve efficiency. By precomputing metadata and optimizing class loading, it ensures fast startup times for JVM and native binary deployments, cutting down on memory usage.</p>
```

Comment:
What's the reason to put performance before developer joy?

Reply:
No good reason. I agree the other way round is probably better, will swap.